schematic/manifest/generator.py

```diff
@@ -751,7 +752,7 @@ def _request_regex_match_vr_formatting(
                 - Upon correct format entry, text will turn black.
                 - If incorrect format is entered a validation error will pop up.
         Input:
-            validation_rules: List[str], defines the validation rules
+            validation_rules: Any(List[str], Dict), defines the validation rules
                 applied to a particular column.
```

Comment:
I know this is the docstring, but this isn't how Any isused for typing. It also doesn't thta type in the function definition. It looks like list[str] is correct.

Reply:
Oh yes, I changed what I was taking in while working on it...and yea I originally thought Any worked like Union... Ill remove this.